src/test/java/se/citerus/dddsample/domain/model/cargo/DeliveryTest.java

```diff
@@ -1,75 +0,0 @@
-package se.citerus.dddsample.domain.model.cargo;
-
-import static org.assertj.core.api.Assertions.assertThat;
-import static se.citerus.dddsample.domain.model.location.SampleLocations.HONGKONG;
-import static se.citerus.dddsample.domain.model.location.SampleLocations.NEWYORK;
-
-import java.util.Date;
-
-import org.junit.Test;
-
-public class DeliveryTest {
-
-  private Cargo cargo = new Cargo(new TrackingId("XYZ"), new RouteSpecification(HONGKONG, NEWYORK, new Date()));
-
-  @Test
-  public void testToSilenceWarnings() {
-    assertThat(true).isTrue();
-  }
-  
-  /*
-  public void testEvensOrderedByTimeOccured() throws Exception {
```

Comment:
I think this is a good example of leaving the code commented instead of deleting it.

`DeliveryTest` has no tests now, and no sign of having tests before!